### PR TITLE
Product and User column tests

### LINKED ISSUE #13: Test that products have a style column

### LINKED ISSUE #21: Test a user has a style-profile, and that they're strings

### PR DESCRIPTION
**Pull request by LHJE**
This PR:
- Adds tests for the style columns in User and Products.

Closes #21 
Closes #13

### CODE REVIEW

File: code/api/package.json [hunk 1]
```diff
@@ -12,7 +12,10 @@
     "start": "nodemon src/index.js --exec babel-node",
     "start:prod": "npm run build:prod && npm run start:server",
     "build:prod": "babel src -s -D -d build",
-    "start:server": "node build/index.js"
+    "start:server": "node build/index.js",
+    "test": "jest",
+    "test:watch": "jest --watch"
```
Comment (Arique1104) [inline code/api/package.json:17]: Kudos on adding your new scripts

File: code/api/src/modules/product/tests/productsQuery.test.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  it('returns a product based on id and only has style info', async(done) => {
+    const response = await request(server)
+      .post('/graphql')
+      .send({query: `{productById(productId:1) { style } }`})
+      .expect(200)
+    expect(response.body).toMatchObject({
+      data:{
+        productById: {
+          style: 'Classy'
+        }
```
Comment (drewwilliams5280) [inline code/api/src/modules/product/tests/productsQuery.test.js:37]: Syntax to remember for future reference

File: code/api/src/config/database.json [hunk 3]
```diff
@@ -7,6 +7,14 @@
...
     "seederStorage": "sequelize"
   },
+  "test": {
+    "username": "postgres",
+    "password": null,
+    "database": "crate_testing",
+    "host": "127.0.0.1",
+    "dialect": "postgres",
+    "seederStorage": "sequelize"
+  },
```
Comment (Arique1104) [inline code/api/src/config/database.json:17]: Great job adding a test database.

File: code/api/src/modules/product/types.js [hunk 4]
```diff
@@ -11,6 +11,7 @@ const ProductType = new GraphQLObjectType({
     name: { type: GraphQLString },
     slug: { type: GraphQLString },
     type: { type: GraphQLInt },
+    style: { type: GraphQLString },
```
Comment (Arique1104) [inline code/api/src/modules/product/types.js:14]: Good catch adding this type here!

File: code/api/src/modules/user/model.js [hunk 5]
```diff
@@ -14,6 +14,12 @@ module.exports = function(sequelize, DataTypes) {
     },
     role: {
       type: DataTypes.TEXT
+    },
+    primaryStyle: {
+      type: DataTypes.STRING
```
Comment (Arique1104) [inline code/api/src/modules/user/model.js:19]: I would like to know the difference between .TEXT and .STRING

Comment (Arique1104) [reply code/api/src/modules/user/model.js:19]: Per convo with @LHJE 
TEXT is for extremely long strings
STRING is a normal string.

**Review by Arique1104 — APPROVED** [review]
Looks great!  Good job team!

**Comment by Arique1104** [comment]
This adds the following:

Tests for product
- This test checks that attribute `style` is  exposed as a valid endpoint.

Test for user
- This test assures that the attributes `primaryStyle` and `secondaryStyle` are exposed as a valid endpoint.

**Pull request merged**